tests/functional/utils.py

```diff
@@ -31,7 +33,7 @@ def get_run_id(secrets, pr_number=None):
     runs = json.loads(r.text)
 
     for run in runs['workflow_runs']:
-        if run['head_sha'] == pr['head']['sha']:
+        if run['head_sha'] == pr['head']['sha'] and run['name'] == CERTIFICATION_CI_NAME:
```

Comment:
There might be multiple workflow runs associate to one PR, such as the new `Workflow Test`. We only want to monitor the result of chart certification run result. Without this line, the script will falsely monitor whichever workflow run was created later, `Workflow Test` in this case.